app/src/main/java/org/wikipedia/talk/TalkTopicsActivity.kt

```diff
@@ -55,12 +60,12 @@ class TalkTopicsActivity : BaseActivity() {
     private lateinit var pageTitle: PageTitle
     private lateinit var invokeSource: Constants.InvokeSource
     private lateinit var notificationButtonView: NotificationButtonView
-    private lateinit var talkTopicsProvider: TalkTopicsProvider
+    private val viewModel: TalkTopicsViewModel by viewModels()
     private var funnel: TalkFunnel? = null
     private var actionMode: ActionMode? = null
     private val searchActionModeCallback = SearchCallback()
     private val disposables = CompositeDisposable()
-    private val topics = mutableListOf<TalkPage.Topic>()
+    private val topics = mutableListOf<ThreadItem>()
```

Comment:
Can the list of topics live in the ViewModel now?

Reply:
Yes!